tests/cases/webroot_test.py

```diff
@@ -45,11 +45,17 @@ def testAccessWebRoot(self):
         emailAddress = self.model('setting').get(SettingKey.CONTACT_EMAIL_ADDRESS)
         self.assertTrue('contactEmail: \'%s\'' % emailAddress in body)
         self.model('setting').set(SettingKey.CONTACT_EMAIL_ADDRESS, 'foo@bar.com')
+
+        brandName = self.model('setting').get(SettingKey.BRAND_NAME)
+        self.assertTrue('<title>%s</title>' % brandName in body)
```

Comment:
I haven't looked at the specs for python's order-of-operations to know whether this is safe, but it would make me more comfortable to be explicit here using parens:

    self.assertTrue(('<title>%s</title>' % brandName) in body)